src/nbt/compound.rs

```diff
@@ -2,59 +2,58 @@ use bytes::{Buf, BufMut, Bytes, BytesMut};
 use crab_nbt::nbt::tag::NbtTag;
 use crab_nbt::nbt::utils::{get_nbt_string, END_ID};
 use derive_more::Into;
-use std::collections::HashMap;
+use std::collections::{hash_map::IntoIter, HashMap};
 
-#[derive(Debug, PartialEq, Clone, Default, Into)]
+use crate::{error::Error, Nbt};
+
+/// Compound of named nbt tags.
```

Comment:
Buttery butter

Reply:
```suggestion
```

Reply:
I know, but it looks nice, when the docs are not empty.
![image](https://github.com/CrabCraftDev/CrabNBT/assets/67533827/5b8f7233-e017-4e9f-b025-8938d18c977e)

Reply:
Lmao... Who cares about docs where they are filled with useless info

Reply:
> Lmao... Who cares about docs where they are filled with useless info

Lmao... Who cares about useless info when the docs look good